src/ZoneServer/World/Actors/Characters/Components/InventoryComponent.cs

```diff
@@ -597,6 +597,8 @@ public InventoryResult Remove(Item item, int amount, InventoryItemRemoveMsg msg)
 
 				Send.ZC_ITEM_REMOVE(this.Character, item.ObjectId, amount, msg, InventoryType.Inventory);
 
+				Send.ZC_ITEM_INVENTORY_DIVISION_LIST(this.Character);
```

Comment:
Is a full refresh of the inventory actually necessary? We're only decreasing an item's amount here, and a refresh is a relatively expensive operation.